bot/cogs/verification.py

```diff
@@ -55,7 +57,11 @@ def mod_log(self) -> ModLog:
     async def on_message(self, message: Message) -> None:
         """Check new message event for messages to the checkpoint channel & process."""
         if message.author.bot:
-            return  # They're a bot, ignore
+            # They're a bot, delete their message after the delay.
+            # But not the periodic ping; we like that one.
+            if message.content != PERIODIC_PING:
```

Comment:
Since this is an `on_message` listener, won't it trigger for all messages - and since this is before the check that ensures it's in the correct channel, delete any messages sent by the bot after 10 sec?

Reply:
Yes you're right. The channel check on l66 needs to come first.